src/main/java/org/ow2/proactive/addons/webhook/Webhook.java

```diff
@@ -34,17 +34,19 @@
  */
 package org.ow2.proactive.addons.webhook;
 
-import java.io.BufferedReader;
-import java.io.InputStream;
-import java.io.InputStreamReader;
-import java.io.OutputStream;
-import java.net.HttpURLConnection;
-import java.net.URL;
-
-import org.ow2.proactive.addons.webhook.exception.UnsuccessfulRequestException;
+import org.ow2.proactive.addons.webhook.service.ApacheHttpClientRequestGetter;
+import org.ow2.proactive.addons.webhook.service.JsonRestApacheRequestService;
+import org.ow2.proactive.addons.webhook.service.JsonStringToHeaderMap;
 
+@SuppressWarnings("WeakerAccess")
 public class Webhook {
 
+    @SuppressWarnings("CanBeFinal")
+    private static WebhookExecutor webhookExecutor = new WebhookExecutor(
```

Comment:
I did it for test-ability,  since the Webhook is basically a static wrapper around the WebhookExecutor so that one does not need to create a new instance in the groovy task. 
In order to test the static wrapper I put the member as static, because I need to change the modifiers in the tests. But it is possible to remove the final modifier in the test. I just decided to do it without final.

What is in your opinion the better approach?